docs/dev/reference/sdks/connectivity.md

```diff
@@ -57,7 +57,51 @@ machine, err := client.New(
 
 {{% /tab %}}
 {{% tab name="TypeScript" %}}
-The TypeScript SDK currently does not support mDNS which means you cannot use it to connect to your machine over a local network.
+
+{{< alert title="Caution" color="caution" >}}
+Using the Typescript SDK over local network or offline requires disabling TLS verification.
+{{< /alert >}}
+
+{{< tabs >}}
+{{% tab name="Command-line" %}}
+
+Restart `viam-server` the `-no-tls` flag.
+
+{{% /tab %}}
+{{% tab name="Configuration" %}}
+
+1. Add `"no_tls": true` to the `"network"` section of your machine's JSON configuration:
+
+   ```json {class="line-numbers linkable-line-numbers" data-line="5"}
+   "network": {
+   "no_tls": true
+   }
+   ```
+
+1. Restart your machine.
+   You can restart your machine by clicking on the machine status indicator in Viam and clicking **Restart**.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Update your connection code by adding `.local` to your host address:
+
+```ts {class="line-numbers linkable-line-numbers" data-line="1,12"}
+const host = "mymachine-main.0a1bcdefgi.local.viam.cloud";
+
+const machine = await VIAM.createRobotClient({
+  host,
+  credentials: {
+    type: "api-key",
+    /* Replace "<API-KEY>" (including brackets) with your machine's API key */ payload:
+      "<API-KEY>",
+    authEntity: "<API-KEY-ID>",
+    /* Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID */
+  },
+  signalingAddress: `http://${host}:8080`,
```

Comment:
As I recall, the signaling address needs to be `http://${host}.local:8080` and the host address doesn't need to be changed. That might be out of date or misrecalled, however. To confirm, have you tried connecting in a no-internet mode with these instructions to confirm it works?

Reply:
Okay the issues I am experiencing appear to be browser-dependent (offline mode seems to be having issues on chrome but works on safari). This seems environmental and I will dig into it; creating a ticket for it.

In the meantime, though, when I _was_ able to get offline mode working, it was by following the instructions that already existed in the TS README. So I would say we should stick with those instructions and _not_ point people to using a `.local.viam.cloud` address, as that will not work.

Reply:
Ok we'll have to pair on this probably because I did try this earlier and I could not get it to work with the .local suffix. This is also a problem if we have it wrongly documented for all the SDKs currently...
 https://deploy-preview-4469--viam-docs.netlify.app/dev/reference/sdks/connectivity/#connect-over-local-network-or-offline